server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -2360,7 +2360,7 @@ public void run() {
         zoo.putPersistentData(zPath, new byte[0], NodeExistsPolicy.SKIP);
 
         if (tabletServerLock.tryLock(lw, lockContent)) {
-          log.debug("Obtained tablet server lock " + tabletServerLock.getLockPath());
+          log.info("Obtained tablet server lock " + tabletServerLock.getLockPath());
```

Comment:
Is it possible to add the lock path for all the locks, or is this special to tservers?